app/Http/Controllers/PublicWikiController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Wiki;
+use App\Http\Resources\PublicWikiResource;
+use App\Http\Resources\PublicWikiCollection;
+
+class PublicWikiController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index(Request $request)
+    {
+        $perPage = $request->query('per_page', null);
+        if ($perPage !== null) {
+            $perPage = intval($perPage);
+        }
+
+        $query = Wiki::query();
+
+        $isFeatured = $request->query('is_featured', null);
+        if ($isFeatured !== null) {
+            $query = $query->where(['is_featured' => boolval($isFeatured)]);
+        }
+
+        $isActive = $request->query('is_active', null);
+        if ($isActive !== null && boolval($isActive)) {
```

Comment:
This seems terrible, but my PHP is not good enough to come up with something better. Is there any more idiomatic way to do this?